### PR TITLE
Anush Manvelyan: Homework about StreamAPI

### PR DESCRIPTION
**Pull request by anushmanvelyan**

### CODE REVIEW

File: homeworks/anush_manvelyan/Maven_TestingQueue/QueueTest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
```
Comment (sirak92) [inline homeworks/anush_manvelyan/Maven_TestingQueue/QueueTest.java:3]: the same comment as for Garik...
You catch the idea but the unit tests should cover the corner cases also

**Review by efixty — APPROVED** [review]
